tests/smoke_test/test_smoke_synapse_connectivity.py

```diff
@@ -0,0 +1,16 @@
+from tests.util.notebook_run_test_case import NotebookRunTestCase
+import json
+
+
+class TestSmokeSynapseConnectivity(NotebookRunTestCase):
+    """
+        This test verifies that external connections from synapse are working properly
```

Comment:
Maybe you can add here which external connection ? KeyVault, DataLake, Failover ....

Reply:
Changed